Framework/Algorithms/inc/MantidAlgorithms/OffspecBackgroundSubtraction.h

```diff
@@ -0,0 +1,45 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
```

Comment:
I know this is a draft PR so you may be changing it, but this license should be 2019